rpcserver.go

```diff
@@ -9,7 +9,6 @@ import (
 	"crypto/tls"
 	"errors"
 	"fmt"
-	xcontext "golang.org/x/net/context"
```

Comment:
This context is being replaced by the stdlib context package (they are the same thing in Go versions which support alias types), so it's fine here.